timeplus-native-jdbc/src/main/java/com/timeplus/data/ColumnLowCardinality.java

```diff
@@ -27,23 +28,39 @@ public class ColumnLowCardinality extends AbstractColumn {
     private final List<Long> indexes;
     private final List<Object> dict;
     private final Long version = 1L;
+    private boolean nested_is_nullable;
 
     public ColumnLowCardinality(String name, DataTypeLowCardinality type, Object[] values) {
         super(name, type, values);
         indexes = new ArrayList<>();
         dict = new ArrayList<>();
         data = ColumnFactory.createColumn(null, type.getNestedTypes(), null);
+        nested_is_nullable = type.getNestedTypes().nullable();
+        if (nested_is_nullable) {
+            dict.add(type.getNestedTypes().defaultValue());
+            dict.add(type.getNestedTypes().defaultValue());
```

Comment:
Add a note here: `If a nested type is nullable, always add two hard dictionary keys in front: [0]: null, [1]: default value`